src/marshmallow_sqlalchemy/convert.py

```diff
@@ -13,6 +14,9 @@
 from .fields import Related, RelatedList
 
 
+_META_KWARGS_DEPRECIATED = LooseVersion(ma.__version__) >= LooseVersion("3.10.0")
```

Comment:
Need to behave differently if `fields.Field(..., metadata=None, ...)` or `fields.Field(..., **metadata)`.

Reply:
s/DEPRECIATED/DEPRECATED

Reply:
Wow.. TIL